docs/getting_started/user-guide.rst

```diff
@@ -203,6 +203,11 @@ variable replacement. A summary table of variables is included below, and then f
    * - singularity_module
      - if defined, add to module script to load this Singularity module first
      - null
+   * - podman_module
```

Comment:
Should podman module be here? The singularity path addition is fantastic - great for security too.